nautobot_golden_config/jobs.py

```diff
@@ -187,7 +187,7 @@ def run(self, data, commit):
             backup_repo.push()
 
 
-class AllGoldenConfig(Job):
+class AllGoldenConfig(Job, FormEntry):
```

Comment:
this job is to run all jobs on 1 device. Not sure I understand why this `FormEntry` is needed on this one.

Reply:
yes agreed, unless you tested this and showed the need for it, not sure why this is needed :)

Reply:
My bad. Was a bit overzealous. Have fixed this.